util/api-config.js

```diff
@@ -0,0 +1,26 @@
+const { verifyChain } = require('./verify-chain.js');
+const avgTxCost = require('./activity-report/avg-tx-cost.js');
+
+module.exports = {
+  '/api/v1/rsk-activity-report/avg-tx-cost': {
+    cacheTtl: 600, // seconds
+    queryDb: avgTxCost.queryDb,
+    queryStringParams: [
+      {
+        name: 'chain',
+        defaultValue: 'rsk_mainnet',
+        verify: verifyChain,
+      },
+      {
+        name: 'blocks',
+        defaultValue: '100',
+        verify: avgTxCost.verifyBlocks,
+      },
+    ],
+    defaultValues: {
+      time: new Date(0),
```

Comment:
for `time` which I think is used for determining how old the cached value is, the *correct* value would be to input the timestamp of when the default values were extracted. e.g. if these values were from 01/12/2022, we use that particular date.

that way, in the rare occasion when the service is down, the front end can determine if it has stale data (and display any warnings accordingly)